Riot/Modules/Room/RoomViewController.m

```diff
@@ -4302,8 +4302,17 @@ - (void)showAdditionalActionsMenuForEvent:(MXEvent*)selectedEvent inCell:(id<MXK
                 
                 [self startActivityIndicator];
                 
+                NSArray<NSString *>* relationTypes = nil;
+                // If it's a voice broadcast, delete the selected event and all related events (only if this feature is supported).
+                if (selectedEvent.eventType == MXEventTypeCustom && [selectedEvent.type isEqualToString:VoiceBroadcastSettings.voiceBroadcastInfoContentKeyType]) {
+                    // Check if the homeserver supports redaction with relations
```

Comment:
I would not check the support of this option here, I will let the SDK check and log a warning until this is supported by the HS